apis/user-java/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:8-jdk-alpine
+
+EXPOSE 8080
+
+# Add a volume pointing to /tmp
+VOLUME /tmp
+
+# The application's jar file
+ARG JAR_FILE=target/swagger-spring-1.0.0.jar
```

Comment:
`ARG` shouldn't be set and if the value (name) of the JAR is always the same, you shouldn't need the ARG at all.